Header header

float64 raw_pos
int16 high_index
int16 low_index
float64 percent_indexed